src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -196,6 +196,20 @@ private void innerTestGetPath(String s) throws IOException {
         Assert.assertTrue(size>0);
     }
 
+    @DataProvider
+    public Object[][] absoluteNames() {
+        return new Object[][] {
+                {"/local/example.txt", "/local/example.txt"},
+                {"/local/file://example.txt", "/local/file:/example.txt"},
+                {"file:///local/example.txt", "/local/example.txt"}
```

Comment:
Add a test case with a non-file URI (such as `gcs://`)

Reply:
I guess that you mean adding the `gs://` example, that is the prefix in GCS. Otherwise it fails because `IOUtils.getPath` does not recognize it as a non-file URI.

Done and test passing.